src/index.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+const tempDisplay = document.getElementById('temp-display');
+const landscapeImg = document.getElementById('landscape-img');
+const cityName = document.getElementById('city-name-display');
+const cityInputBox = document.getElementById('city-name');
+
+const state = {
+  temp: 80,
+  city: cityName.textContent,
+  color: 'rgb(248, 163, 163)',
+};
+
+// Set default values
+tempDisplay.textContent = `${state.temp}°`;
+tempDisplay.style.color = state.color;
```

Comment:
I would recommend wrapping these in a function and calling the function at the bottom of the file where `document.addEventListener...` is. That helps us be consistent with how we have our code organized by keeping functions definitions and code we are immediately executing sectioned together.